extension/background.js

```diff
@@ -437,7 +437,7 @@ router["/tabs/by-title/*"] = {
   // TODO: date
   async readlink({path}) { // a symbolic link to /tabs/by-id/[id for this tab]
     const parts = path.split("_"); const tabId = parts[parts.length - 1];
-    return { buf: "../by-id/" + tabId };
+    return { buf: "../../tabs/by-id/" + tabId };
```

Comment:
It seems that the length of the buffer causes the issue. A longer string fixes the problem